drake/systems/plants/RigidBodyFrame.h

```diff
@@ -3,9 +3,18 @@
 
 class DLLEXPORT_RBM RigidBodyFrame {
 public:
+  RigidBodyFrame(const std::string& _name, const std::shared_ptr<RigidBody>& _body, const Eigen::Matrix4d& _Ttree)
```

Comment:
I guess I feel like I cleaned up a lot of this stuff in the KinematicsCache pull request and would not like to see the use of Matrix4d and the name Ttree 'reintroduced' (to me at least) right away.